projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -1,24 +1,28 @@
 ## Global Docker image parameters
 ## Please, note that this will override the image parameters, including dependencies, configured to use the global value
-## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+## Current available global Docker image parameter(s): imageRegistry
 ## Set global.imageRegistry if you use docker proxy to the proxy prefix for versatiledatakit.
 ##
 global: {}
 #   imageRegistry: myRegistryName
-#   imagePullSecrets: # TODO enable setting imagePullSecrets so that clients can use any docker repos easier (clients can still set it up manually)
 #     - myRegistryKeySecretName
 
 # Image configuration of the Control service image. Generally those should be left as it is.
 image:
-   registry: registry.hub.docker.com/versatiledatakit
-   repository: pipelines-control-service
-   tag: "1f57f0e"
-
-   ## Specify a imagePullPolicy
-   ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
-   ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
-   ##
-   pullPolicy: IfNotPresent
+  registry: registry.hub.docker.com/versatiledatakit
+  repository: pipelines-control-service
+  tag: "1f57f0e"
+  ## Image registry types. Your account need to have pull permissions over the registry. The supported values are:
+  ## - "generic" : Dockerhub or Harbor type registries
+  ## TODO Support will be added for more later
+  registryType: generic
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  registryUsernameReadOnly:
+  registryPasswordReadOnly:
```

Comment:
Can you elaborate? 
Should we assume that the deployment builder image and pipelines-control-service images are always in the same repo?

Reply:
We don't support pulling the builder image from a private repo at the moment. https://vmware.slack.com/archives/C023WRM7LAC/p1667409141423119?thread_ts=1666624344.264629&cid=C023WRM7LAC

But even if we did I don't understand your point?

Reply:
Your question was 

> Should we assume that the deployment builder image and pipelines-control-service images are always in the same repo?

My point is, it is possible that a VDK user could use a different registry/repo (it may be publically accessible or not) for builder image because they would customize it. 

This is how they'd build "DevOps plugin" as showed in one our talks - which I encourage you to watch if you have not - https://www.youtube.com/watch?v=TSh9KI8WAjI